db/migrate/20170509210708_add_use_notifications_to_info_request.rb

```diff
@@ -0,0 +1,5 @@
+class AddUseNotificationsToInfoRequest < ActiveRecord::Migration
+  def change
+    add_column :info_requests, :use_notifications, :boolean
```

Comment:
> I don't think a database default would affect that as it would only take effect once the record is created.

Hmm, I did spend a bit of time trying to make it work, but it did seem that rails was also defaulting it. I'll try applying a migration again over this and see what happens. I agree it would make things clearer all round.